user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -58,6 +60,18 @@ Each row in a query corresponds to an item in the grid.
 > [!TIP]
 > For an example of how to configure a GQI query that can be used as data input for a grid component, see [Tutorial: Getting started with the grid component](xref:Tutorial_Apps_Grid).
 
+## Using the grid as a selector
+
+You can use a grid component as an interactive selector, where users click grid items to control what is displayed in other components.
+
+In the example below, a grid filters a table component based on job state. By selecting a grid item, users can quickly switch between active, upcoming, completed, or all jobs.
+
+![Grid as a selector](~/user-guide/images/Grid_as_Selector.gif)<br>*Grid component and table component in DataMiner 10.5.7*
+
+To use a grid as a selector:
+
+1.
```

Comment:
While editing an app:

1. Select an item in your grid.
2. Update the configuration of the consumer and link it to the selection (using either the _Edit panel_ or the _Link to data popup_).